steps/discover-repos.rb

```diff
@@ -92,9 +114,9 @@
       topics: i[:topics]
     }
     puts "Found #{i[:full_name].inspect} GitHub repo ##{found.count} \
-(#{i[:forks_count]} forks, #{i[:stargazers_count]} stars)"
+(#{i[:forks_count]} forks, #{i[:stargazers_count]} stars) with license: #{i[:license][:name]}"
```

Comment:
@Orillio why do we need their "names"? I suggest to rely only on the `:key` and ignore the name